util/helper_scripts_for_mongodb/requirements.txt

```diff
@@ -0,0 +1,3 @@
+```
```

Comment:
Is this line supposed to be here?

Reply:
It gets automatically added by pre commit